Apps/CocoaPods-FCM/src/View/Appium.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import SwiftUI
+
+extension View {
+    // Centralized way to set the identifier on a View so Appium can find the view.
+    func setAppiumId(_ viewId: String) -> some View {
+        accessibility(identifier: viewId)
+    }
+}
```

Comment:
curious to know, why don't we directly use `accessibility(identifier: viewId)`

There is no other logic inside this method, why not just use this method directly from wherever we are calling `setAppiumId`.

Reply:
2 reasons. 

1. Documentation of the code. If someone was reading the code and saw `accessibility(identifier: "Login button")`, they might wonder why we are calling that function. Seeing `setAppiumId` gives a reason for why the code is called. 
2. I am not yet sure if `accessibility(identifier: viewId)` is the correct way to add appium view tagging. If I am wrong, it's 1 line of code to fix the code to work for appium.